.ci/common/test.sh

```diff
@@ -110,7 +110,16 @@ run_oldtests() {
 }
 
 run_single_includes_tests() {
-  ${MAKE_CMD} -C "${BULID_DIR}" check-single-includes
+  ulimit -c unlimited
+  if ! ${MAKE_CMD} -C "${BUILD_DIR}" check-single-includes; then
+    asan_check "${LOG_DIR}"
+    valgrind_check "${LOG_DIR}"
+    check_core_dumps
```

Comment:
@ZyX-I I know what you commented, but you could see the tests on TravisCI passed by adding these script. It seems that the reason the tests did not pass was the test does not output anything. I would like to have confirmed that.
 If so, what kind of output is appropriate to let the tests passed?